AGENTS.md

```diff
@@ -0,0 +1,60 @@
+# AGENTS.md
+
+## Dev environment tips
+
+```bash
+# Setup
+npm install && composer install
+npm run wp-env status   # Check if wp-env running
+npm run wp-env start    # Start WordPress environment
```

Comment:
`npm run wp-env stop` too?

Reply:
I don't have a strong opinion. The idea here is not to teach the available commands to the agent, but to be explicit about checking the status before assuming wp-env is not running and trying to start it. I'll reword this 👍